app/views/dump.py

```diff
@@ -25,22 +24,32 @@ def dump_debug():
     return questionnaire_store.serialise()
 
 
+@dump_blueprint.route('/dump/routing', methods=['GET'])
+@login_required
+@role_required('dumper')
+def dump_routing():
+    response = {'routing': 'routing-goes-here'}
```

Comment:
🤔

Reply:
That was a placeholder... Am I right in saying that this is now all encapsulated by the `/dump/submission/` endpoint?

Reply:
No, we have a `/dump/debug` endpoint, but that just dumps the contents of the questionnaire store. It would be useful to be able to view the routing path per section as an endpoint.